README.md

```diff
@@ -287,6 +289,8 @@ The summary includes the following fields:
 * `sampleStatus2ED10Time`
 * `sampleStatus1UR10`
 * `sampleStatus1UR10Time`
+* `sampleStatus1UR90`
```

Comment:
Please add the other samples in here, too.

Reply:
done